src/Facility.AspNetCore/Facility.AspNetCore.csproj

```diff
@@ -1,7 +1,8 @@
-<Project Sdk="Microsoft.NET.Sdk">
+<Project Sdk="Microsoft.NET.Sdk.Web">
 
   <PropertyGroup>
-    <TargetFramework>netstandard2.0</TargetFramework>
+    <TargetFramework>net6.0</TargetFramework>
```

Comment:
Maybe this is prohibitive? Making it work with a netstandard target didn't seem trivial at first blush. Giving it another whirl.
https://github.com/dotnet/AspNetCore.Docs/issues/22048
https://docs.microsoft.com/en-us/aspnet/core/fundamentals/target-aspnetcore

Reply:
I don't mind not supporting .NET Standard as long as we support the target frameworks that aren't EOL (currently .NET Core 3.1, .NET 5, .NET 6).